daemon/loadbalancer.go

```diff
@@ -130,7 +140,8 @@ func (d *Daemon) svcAdd(
 	// of the backends when listing services. This step will go away once
 	// we start acquiring backend IDs in this module.
 	for i, be := range svc.BES {
-		id, err := service.LookupBackendID(be.L3n4Addr)
+		var id loadbalancer.BackendID
+		id, err = service.LookupBackendID(be.L3n4Addr)
 		if err != nil {
 			scopedLog.WithField(logfields.BackendName, be.L3n4Addr).WithError(err).
 				Warning("Unable to lookup backend ID")
```

Comment:
Why do we continue if err != nil? If err is not nil for index 0 but is nil for index 1, why is that ok?

Reply:
This is unrelated to my PR, and this is not critical, as the retrieved IDs are used only for `cilium service list`.

Anyway, I have in my LB refactoring TODO to get rid of this.

Reply:
@brb it is related with your PR because the `err` in line 144 can be the same of line `99`

Reply:
Oops, nice catch. Fixed.